packages/web/src/components/hustler/ConfigurationControls.tsx

```diff
@@ -115,11 +115,22 @@ const ConfigurationControls = ({
         .toString(16)
         .padStart(4, '0');
 
-    let bitmask = 11110110;
+    // Bitmask controls configuration options we're allowed to set 
```

Comment:
add `1000` to `bitoptions` on line 99 in order to enable order

Reply:
Still not persisting on-chain after latest fix. If WEAPON is slot 0, I'm trying to pass in same ordering as contract with no apparent change. Also tried as last item in array with no effect.